mbed_lstools/lstools_ubuntu.py

```diff
@@ -15,297 +15,14 @@
 limitations under the License.
 """
 
-import re
-import subprocess
+from lstools_linux_generic import MbedLsToolsLinuxGeneric
 
-from lstools_base import MbedLsToolsBase
 
-
-class MbedLsToolsUbuntu(MbedLsToolsBase):
+class MbedLsToolsUbuntu(MbedLsToolsLinuxGeneric):
```

Comment:
The only reason is this comment https://github.com/ARMmbed/mbed-ls/issues/25#issuecomment-151058043 - where @PrzemekWirkus stated that Ubuntu is the main testing platform.